lib/rtcpeerconnection/safari.js

```diff
@@ -21,8 +20,6 @@ function SafariRTCPeerConnection(configuration) {
   util.interceptEvent(this, 'signalingstatechange');
   util.interceptEvent(this, 'track');
```

Comment:
We don't need to `interceptEvent` or call `addEventListener` below anymore, do we?